### ISSUE TITLE
ISA 51:21b metaphor

### ISSUE THREAD

**Issue by kim-putter** [Door43/udb-en#164]
""It is as though I had taken that cup of bitter liquid from your hands;
I will not be angry with you and cause you to drink it any longer, to suffer anymore."

This continues a metaphor that began in 51:17. In the comment section, there is a discussion where Jim changed the cup so that it did not contain a bitter liquid, but rather wine. Here in verse 22, it still refers to a bitter liquid. This probably needs changed but I'll leave the research to you.

**Comment by TomWarren** [comment]
The UDB of ISA 51:17-22 now reads: 
\v 17 You people of Jerusalem, wake up!
\q2 You have experienced Yahweh severely punishing you.
\q1 Yahweh has caused you to suffer much
\q2 and to experience disaster.
\q1 Yahweh gave you this cup to drink, but this was Yahweh's cup! 
\q2 He forced you to drink every drop out of that large cup, down to the bottom of the bowl. And that drink made you  stagger like you  had drunk strong wine. This strong wine represents the anger of God, and you are to drink all of it!
\q1
\v 18 Now you do not have any children 
\q1 who are able to take your hand and guide you. You cannot help yourself.

\s5
\q1
\v 19 You have experienced these disasters:
\q2 Your country has become desolate;
\q1 your cities have been destroyed;
\q2 many people have died from hunger;
\q2 many people have been killed by your enemies' swords.
\q1 Now there is no one to weep with you and sympathize with you.
\q1
\v 20 Your children have fainted and lie at the corner of every street;
\q2 they are as helpless as an antelope that has been caught in a net.
\q1 What has happened to them is because Yahweh has been very angry with them;
\q2 he has rebuked them severely.
\b

\s5
\q1
\v 21 So now, you people who have suffered much,
\q1 you who were acting as if you were drunk because you were drinking from his cup. 
\q2 But in truth, it is not because you have drunk out of a cup filled with wine, but because of something else that Yahweh will do. 
\q1
\v 22 Yahweh, your Lord and your God,
\q2 the one who argues your case to defend you, he says this:
\q1 "It is as though I had taken away that cup that I gave you, that cup from which you drank, and that made you nearly fall down like one who is drunk with wine. I took it away and you will not drink of it ever again. 
\q2 I will not be angry with you and cause you to drink from that cup of my anger any longer, or to suffer anymore. What I did to you is represented by a cup that holds my anger toward you, and I made you experience my anger, I made you drink from that cup. But now I promise that have I taken that cup, which represented my anger, and I have taken it away from you.
